IBX-3118: Updated note swiftmailer (#1671)
ezsystems/developer-documentation: docs/guide/form_builder/customize_email_notifications.md

Some email providers require a sender address to be set, so to avoid unsent emails when using Form Builder,
it is recommended to configure `sender_address` in `config/packages/swiftmailer.yaml`.
This email acts as a sender and return address for all bounced messages.

!!! note

>     Since November 2021 the Swift Mailer is no longer supported and the integration with Symfony was deprecated in Symfony 6.0.
>     The Swift Mailer will be replaced with the Symfony Mailer.

Reviewer: ```suggestion
    Since November 2021 the Swift Mailer is no longer supported and the integration with Symfony is deprecated in Symfony 6.0.
    The Swift Mailer got replaced by the Symfony Mailer.
```
As it already happened last year we can skip future tens
